.github/workflows/ci.yml

```diff
@@ -77,4 +77,4 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: clippy
-          args: -- -D warnings
+          args: --tests -- -D warnings
```

Comment:
So when I originally looked at #77, I thought we were adding it in addition to the regular targets, but I think what this actually does is only runs clippy on tests (according to the following documentation from `cargo check --help`):

```
Target Selection:
      --lib               Check only this package's library
      --bins              Check all binaries
      --bin [<NAME>]      Check only the specified binary
      --examples          Check all examples
      --example [<NAME>]  Check only the specified example
      --tests             Check all test targets
      --test [<NAME>]     Check only the specified test target
      --benches           Check all bench targets
      --bench [<NAME>]    Check only the specified bench target
      --all-targets       Check all targets
```

Are we sure that this is still running clippy over lib and bin targets? Do we actually just want `--all-targets`?

Reply:
oh! somehow I always thought `--test` does what `--all-targets` actually does! I thought it was inclusive ... thanks for pointing that out!